README.md

```diff
@@ -157,14 +157,17 @@ RUN set -ex; \
         git \
     ; \
     \
-    composer \
+
+# needed to run plugins when using containers
+ENV COMPOSER_ALLOW_SUPERUSER=1
+
+RUN composer \
```

Comment:
do not split the line, it will create one more docker layer
please put the ENV line before the RUN line

Reply:
You're right. 
> Can you also update the README.md example ?
What were you referring to here then?

Reply:
I added one new comment
Basically you have nothing to change, only one line to add after FROM